src/luarocks/fs/lua.lua

```diff
@@ -11,16 +11,16 @@ local dir = require("luarocks.dir")
 local util = require("luarocks.util")
 local path = require("luarocks.path")
 
-local socket_ok, zip_ok, unzip_ok, lfs_ok, md5_ok, posix_ok, _
-local http, ftp, lrzip, luazip, lfs, md5, posix
+local socket_ok, zip_ok, unzip_ok, lfs_ok, digest_ok, posix_ok, _
+local http, ftp, lrzip, luazip, lfs, digest, posix
 
 if cfg.fs_use_modules then
    socket_ok, http = pcall(require, "socket.http")
    _, ftp = pcall(require, "socket.ftp")
    zip_ok, lrzip = pcall(require, "luarocks.tools.zip")
    unzip_ok, luazip = pcall(require, "zip"); _G.zip = nil
    lfs_ok, lfs = pcall(require, "lfs")
-   md5_ok, md5 = pcall(require, "md5")
+   digest_ok, digest = pcall(require, "digest")
```

Comment:
How is it possible that digest_ok is false?

Reply:
@Totktonada and @rosik insist that we don't break compatibility with regular lua. I don't see any major reason to do so, but anyway...